release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -1789,6 +1789,16 @@ Workaround: Use OpenShift OAuth for authentication. (link:https://bugzilla.redha
 *  The China (Nanjing) and UAE (Dubai) regions of Alibaba Cloud International Portal accounts do not support installer-provisioned infrastructure (IPI) installations. The China (Guangzhou) and China (Ulanqab) regions do not support a Server Load Balancer (SLB) if using Alibaba Cloud International Portal accounts and, therefore, also do not support IPI installations.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=2048062[*BZ#2048062*])
 
+* There is currently a known issue. The *Knative Serving - Revision CPU, Memory, and Network usage* and *Knative Serving - Revision Queue proxy Metrics* dashboards are visible to all the namespaces, including those that do not have Knative services. They should show only with namespaces that have Knative services. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2056682[BZ#2056682])
+
+* There is currently a known issue. In the developer perspective, the *Observe* dashboard opens for the most recently viewed workload rather than the one you selected in the *Topology*. This issue happens because the session prefers the redux store instead of the query parameters in the URL. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2052953[BZ#2052953])
+
+* There is currently a known issue. The *ProjectHelmChartRepository* custom resource (CR) does not show up in the cluster because the API schema for this CR has been merged but has not been initialized in the cluster yet. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2054197[BZ#2054197])
+
+* There is currently a known issue. While running high-volume pipeline logs, the auto-scroll functionality does not work and logs are stuck showing older messages. This happens because running high-volume pipeline logs generates a large number of calls to the `scrollIntoView` method. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2014161[BZ#2014161])
```

Comment:
```suggestion
* There is currently a known issue. While running high-volume pipeline logs, the auto-scroll functionality does not work and logs are stuck showing older messages. This issue happens because running high-volume pipeline logs generates a large number of calls to the `scrollIntoView` method. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2014161[BZ#2014161])
```